mmedit/datasets/pipelines/augmentation.py

```diff
@@ -546,6 +596,29 @@ def __repr__(self):
         return self.__class__.__name__ + f'hue_range={self.hue_range}'
 
 
+@PIPELINES.register_module()
+class ColorJitter:
+    """An interface for torch color jitter so that it can be invoked in
+    mmediting pipeline."""
+
+    def __init__(self, **kwargs):
+        self.transform = transforms.ColorJitter(**kwargs)
+
+    def __call__(self, results):
+        # img is bgr
+        img = results['gt_img'][..., ::-1]
```

Comment:
I suggest that we pass an additional argument `to_rgb` so that the function would skip this step if the input image is already an RGB image.